app/utils/rows/RowBuilder.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.rows
+
+import java.time.format.DateTimeFormatter
+
+import models.{Address, UserAnswers}
+import play.api.i18n.Messages
+import uk.gov.hmrc.viewmodels.SummaryList.{Action, Key, Row, Value}
+import uk.gov.hmrc.viewmodels.{Content, Html, MessageInterpolators}
+
+trait RowBuilder {
+
+  implicit val messages: Messages
+  val userAnswers: UserAnswers
+  val dateFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM yyyy")
+
+  private[utils] def yesOrNo(answer: Boolean): Content =
+    if (answer) {
+      msg"site.yes"
+    } else {
+      msg"site.no"
+    }
+
+  private[utils] def toRow(msgKey: String, content: Content, href: String)(implicit messages: Messages): Row = {
+    val message = MessageInterpolators(StringContext.apply(s"$msgKey.checkYourAnswersLabel")).msg()
+    Row(
+      key     = Key(message, classes = Seq("govuk-!-width-one-half")),
+      value   = Value(content),
+      actions = List(
+        Action(
+          content            = msg"site.edit",
+          href               = href,
+          visuallyHiddenText = Some(msg"site.edit.hidden".withArgs(message))
+        )
```

Comment:
Add ids to change links by including 'attributes = Map("id" -> "someID")' to the Action

Reply:
Added.